source/views/menus/dev-bonapp-picker.js

```diff
@@ -73,3 +73,7 @@ export class BonAppPickerView extends React.PureComponent<Props, State> {
 		)
 	}
 }
+BonAppPickerView.navigationOptions = {
+	title: 'BonApp',
+	tabBarIcon: TabBarIcon('cog'),
```

Comment:
There's no tab bar here anymore

Reply:
So... this is still being used in the menus. I'm needing to have it set here so that I can have a title when it's launched from developer settings, and I need to set the tabBarIcon here because we're now setting the title here 🤷‍♂️

Reply:
Just take it out of the Menus section :p

Reply:
Alrighty!